integration-test/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitSpecificIntegrationTestBase.java

```diff
@@ -2944,39 +2932,78 @@ void put_WhenOneOfTransactionsIsDelayed_ShouldBeCommittedWithoutBlocked() throws
 
   @Test
   @EnabledIf("isGroupCommitEnabled")
-  void put_WhenTheOtherTransactionAbortDueToConflict_ShouldBeCommittedWithoutBlocked()
-      throws Exception {
+  void put_WhenTheOtherTransactionsFails_ShouldBeCommittedWithoutBlocked() throws Exception {
```

Comment:
This is a new test case, although it's a bit similar to `put_WhenTransactionFailsDueToConflict_ShouldBeAbortedWithoutBlocked()` below.